src/main/java/codeu/controller/RegisterServlet.java

```diff
@@ -47,8 +48,9 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     @Override
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException, ServletException {
 
-	String username = request.getParameter("username");
-	String password = request.getParameter("password");
+		String username = request.getParameter("username");
```

Comment:
Indention here should be 8 spaces

Reply:
Yes, but for some reason on my ide it shows up on the same indentation, so it could be github?

Reply:
Actually, I think I know why..hold on...